### ISSUE TITLE
Quaterman

### ISSUE THREAD

**Issue by mplain** [db0/Doomtown-Reloaded-OCTGN#43]
Quaterman ignores the mad scientist I target, asking who's gonna invent it, and then it enters play at my home regardless of where the MS was.

**Comment by mplain** [comment]
Seems to be fixed.

**Comment by db0** [comment]
yeah but not the comes into play at the MS' location. Not easy to do that.
